mw-webapp/src/service/auth/logIn.ts

```diff
@@ -1,7 +1,7 @@
 import {signInWithRedirect} from "firebase/auth";
 import {auth, provider} from "src/firebase";
 
-export const handleLogIn = async () => {
+export const logIn = async () => {
```

Comment:
What is the reason to change function names? They are still handlers in my opinion. handleLogIn -> signInWithRedirect, looks better than logIn -> signInWithRedirect

Reply:
handle is too general word. So we try don't use it for specific functions. 
Login is well

Reply:
the main question here does it (using word handle) make code more readable. If no - we should not use it

Reply:
I think try\catch block can be considered handling?

Reply:
let's skip check services to errors, we will improve it latere